src/components/SearchResults/ExactMatch/Gene/index.js

```diff
@@ -58,9 +60,8 @@ ListOfProteins.propTypes = {
   gene: string,
 }; */
 const ExactGeneMatchWrapper = ({ matches, gene }) => (
-  <div className={f('callout', 'warning', 'margin-bottom-medium')}>
-    <span className={f('icon', 'icon-common')} data-icon="&#xf35a;">
-      {' '}
+  <Callout type="warning" customIcon="icon-arrow-alt-circle-right">
```

Comment:
```suggestion
  <Callout type="warning" icon="icon-arrow-alt-circle-right">
```

if accepting [this suggestion](https://github.com/ProteinsWebTeam/interpro7-client/pull/567#discussion_r1471279716).